### ISSUE TITLE
Push button does not enable bluetooth

### ISSUE THREAD

**Issue by antonioli86** [NebraLtd/hm-config#191]
- [x ] I understand that this is only to report software bugs and **not** for customer support. Please use [nebra.io/helium-support](https://nebra.io/helium-support).

**Describe the bug**
Bluetooth button not working.
As described in the manual, in order to enable the Bluetooth we must press the back button for 10 seconds however nothing happens. Workaround is to turn off and turn on - since bluetooth is enabled for some minutes after boot.

Other people experiencing this issue:
https://www.reddit.com/r/NebraMiner/comments/thrivl/my_experience_with_4_rock_pis_so_far/

**To Reproduce**
1. Press button for 10-15 seconds

**Expected behavior**
Bluetooth enabled and visible in Helium app

**Screenshots**
N/A

**Desktop (please complete the following information):**
N/A

**Smartphone (please complete the following information):**
 - Device: OnePlus Nord
 - OS: Android 11
 - Helium app v. 4.0.1

**Additional context**
Model of unit: Nebra HNT Indoor Hotspot Miner - RockPi version 868 MHz (SKU: NBR-0063) - NEBHNT-HHRK4-868

**Comment by shawaj** [comment]
Ping @NebraLtd/developers

**Comment by antonioli86** [comment]
> @antonioli86 I'm moving this over to hm-config repo. We will investigate.
> 
> Can you create a ticket at https://support.nebra.com with a picture of the label on the device? And reply back with the ticket number?

Created yesterday - ticket (#350112).

**Comment by MuratUrsavas** [comment]
It might now be directly related but looks like we have a regression in here:

https://github.com/NebraLtd/hm-config/commit/822ed2aaad4a70527c2ed842edb2864bb0b2d926

@marvinmarnold, as per @pritamghanghas's changes on hm-config, we're not using synchronous sleeps and infinite loops for polling anymore. It is not directly impacting Bluetooth advertisement on startup, but most probably interferes with the button press. Could you take a look at it?

**Comment by pritamghanghas** [comment]
@MuratUrsavas those are different threads. Shouldn't interfere.

**Comment by antonioli86** [comment]
Hi,
Just to inform - after interaction with Nebra Support in ticket (#350112) - "ROCK Pi Indoor green light is off" - now its working and Bluetooth is enabled after push button.

**Comment by MuratUrsavas** [comment]
> those are different threads. Shouldn't interfere.

@pritamghanghas You're right, missed that point. Probably I've been mislead by the log output. Couldn't see the advertisement startup in the logs.

@antonioli86 glad your problem has been resolved. We'll keep this case open until we sort out the effects of the commit I've mentioned.

**Comment by ymc-frgu** [comment]
I have the same problem on one of my devices. I could resolved it by rebooting - luckily the miner enables bluetooth on start up for a short amount of time.

But its worse on another miner of mine. There is no bluetooth at all - not even on start up. So i'm not able to connect and get it up running. 
Ticket #351429

**Comment by shawaj** [comment]
> I have the same problem on one of my devices. I could resolved it by rebooting - luckily the miner enables bluetooth on start up for a short amount of time.
> 
> But its worse on another miner of mine. There is no bluetooth at all - not even on start up. So i'm not able to connect and get it up running. Ticket #351429

@ymc-frgu just leave it online on Ethernet and it'll pull in the latest update with the new firmware
